### PR TITLE
Build presto gateway images

### PR DESCRIPTION
**Pull request by fanglu-wish**
Added dockerfile, bootstrap scripts and docker compose yaml file

### CODE REVIEW

File: gateway-ha/gateway-ha-config.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 requestRouter:
-  port: 8080
+  port: 9091
   name: prestoRouter
   historySize: 1000
 
 dataStore:
-  jdbcUrl: jdbc:mysql://127.0.0.1:3306/prestogateway
+  jdbcUrl: jdbc:mysql://xxx.xxx.xxx.xxx:3306/prestogateway
```
Comment (yizhu-wish) [inline gateway-ha/gateway-ha-config.yml:7]: Can we use ENV variable for password and host?
Even better, consider using config within volume mounted.

Comment (fanglu-wish) [reply gateway-ha/gateway-ha-config.yml:7]: This is for local testing. Will use k8s volume mount to add yams files

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Pull Base Image
+FROM openjdk:11
+
+# Set Maintainer Details
+MAINTAINER Fang Lu <fanglu@wish.com>
```
Comment (yizhu-wish) [inline Dockerfile:5]: use LABEL maintainer="fanglu@wish.com"

File: bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+java -jar target/gateway-ha-1.8.2-jar-with-dependencies.jar server gateway-ha-config.yml
```
Comment (yizhu-wish) [inline bootstrap.sh:3]: If you plan to use config from mounted volume, maybe accept parameter for its full path?

**Pull request merged**